### ISSUE TITLE
Upload image with react-native-image-crop-picker

### ISSUE THREAD

**Issue by MrLibya** [axios/axios#3014]
I'm getting [Error: Network Error] when i try to upload the image, the code will explain

**Example Code**
```js
ImagePicker.openPicker({
            width: 300,
            height: 400,
            cropping: true,
            forceJpg:true,
            mediaType:'photo'
          }).then( async (image) => {
            try {
                //console.log(image);
                var myImage = {
                    uri:image.path,
                    //uri:image.path,
                    name: 'profile_pic.jpeg',
                    type: image.mime, // or photo.type
               };

                //var test = {uri:image.path,type:image.mime,name:'MY_IMAGE'};
                let response = await APIUpdateProfile(myImage);
                console.log(response);
            } catch (error) {
                showMessage({
                    message: error.message,
                    type: "danger",
                    titleStyle:{fontSize:18}
                });
            }
          }).catch(() => { });

// from another file
export async function APIUpdateProfile(data){
    
    try{
        const options = {
            headers: {
                Accept: "application/json",
                "Content-Type": "multipart/form-data"
            }
        };

        const form_data = new FormData();
        form_data.append('image', data);
        console.log(form_data);
        const res = await axios.post(c.UPDATE_PROFILE, form_data,options);
        return res.data;
    }catch (e) {
        console.log(e);
        throw handler(e);
    }
}
```

**the output:**
```
{"_parts": [["image", [Object]]]}
[Error: Network Error]
```

When i tried with postman it worked
https://prnt.sc/sy18ar

**Comment by MrLibya** [comment]
My php api headers:
```php
header("Access-Control-Allow-Origin: *");
header("Access-Control-Allow-Headers: *");
header("Access-Control-Allow-Methods: POST, GET, OPTIONS");
header("Content-Type: multipart/form-data");
```

**Comment by chinesedfan** [comment]
A quick solution is that checking **url** and **request headers** and comparing with other ways when you can get the right response. If you confirm that axios sends wrong url/headers, feel free to remind me to reopen.

**Comment by FaycalBorsali** [comment]
@MrLibya , if the problem persists, this thread can help you ([link to the comment with the answer](https://github.com/facebook/flipper/issues/993#issuecomment-619823916))

**TL;DR** you might need to manually change the `FLIPPER_VERSION` ... and that's it !
